### PR TITLE
UIORGS-84 Add translatable payment methods options

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Add  translatable payment methods options and labels
- enum  payment methods options - [schema](https://github.com/folio-org/acq-models/blob/master/mod-orgs/schemas/account.json#L25)

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Accounts/AccountsForm.js [hunk 1]
```diff
@@ -110,13 +101,10 @@ class AccountsForm extends Component {
...
-                  component={Select}
-                  dataOptions={paymentMethodDD}
-                  fullWidth
+                <FieldSelect
+                  dataOptions={PAYMENT_METHOD_OPTIONS_FOR_SELECT}
                   label={<FormattedMessage id="ui-organizations.accounts.paymentMethod" />}
                   name={`${elem}.paymentMethod`}
-                  placeholder=" "
                   required
                   validate={[Required]}
```
Comment (aliaksei-chumakou) [inline src/Accounts/AccountsForm.js]: I think this isn't required if you use common component

File: src/Accounts/AccountsView.js [hunk 2]
```diff
@@ -13,22 +15,30 @@ const printKeyValue = (label, val, isRequire) => {
...
-);
+const renderAccount = (val, key) => {
+  const paymentMethod = get(val, 'paymentMethod');
+
+  return (
+    <Row
+      className={css.account}
+      key={key}
+    >
+      {printKeyValue('name', get(val, ['name'], ''), false)}
```
Comment (aliaksei-chumakou) [inline src/Accounts/AccountsView.js]: I don't think we need to use `printKeyValue` at all

File: src/common/constants/organization.js [hunk 3]
```diff
@@ -15,7 +15,18 @@ export const ORGANIZATION_STATUS_LABELS = {
...
+  eft: 'EFT',
+  depAccount: 'Deposit Account',
+  physicalCheck: 'Physical Check',
+  bankDraft: 'Bank Draft',
+  internalTransfer: 'Internal Transfer',
+  other: 'Other',
+};
+
+export const ORGANIZATION_PAYMENT_METHOD_FOR_FILTER = {
+  cash: 'Cash',
```
Comment (aliaksei-chumakou) [inline src/common/constants/organization.js]: If this is a subset of the full payment methods list please use constants, don't copy-paste string values.

File: src/Accounts/AccountsView.js (hunk 2 above)
Comment (AndreiShumski2) [reply src/Accounts/AccountsView.js]: Yes. I know. I added `PaymentMethod` only and didn't touch the rest code.

File: src/common/constants/organization.js [hunk 4]
```diff
@@ -15,7 +15,18 @@ export const ORGANIZATION_STATUS_LABELS = {
 
 export const ORGANIZATION_PAYMENT_METHOD = {
   cash: 'Cash',
-  card: 'Credit Card/P-Card',
+  card: 'Credit Card',
```
[the remark this answers is not in the document]
Comment (AndreiShumski2) [reply src/common/constants/organization.js]: Added consts to [stripes-acq-components](https://github.com/folio-org/stripes-acq-components/pull/70)

File: src/Accounts/AccountsView.js (hunk 2 above)
Comment (aliaksei-chumakou) [reply src/Accounts/AccountsView.js]: well, it's green in github and marked as "touched") I see 2 options - change only lines you need or refactor to not use `printKeyValue`

Comment (AndreiShumski2) [reply src/Accounts/AccountsView.js]: Done

**Pull request merged**